courses/machine_learning/intro_to_tensorflow/4_keras_functional_api.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
<ul><li>Remove the following comments which don't add more to what the variable and function names are describing:</li><li class="ql-indent-1">&nbsp;&nbsp;# Compute Euclidean distance</li><li class="ql-indent-1">  # Add engineered features</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/576/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>